chainer_chemistry/dataset/splitters/random_splitter.py

```diff
@@ -0,0 +1,17 @@
+import numpy
+from chainer_chemistry.dataset.splitters.base_splitter import BaseSplitter
+
+
+class RandomSplitter(BaseSplitter):
+    def split(self, dataset, seed=None, frac_train=.8, frac_valid=.1,
+              frac_test=.1):
+        numpy.testing.assert_almost_equal(frac_train + frac_valid + frac_test,
+                                          1.)
+        if seed is not None:
+            numpy.random.seed(seed)
+        perm = numpy.random.permutation(len(dataset))
```

Comment:
Setting numpy random seed may have side effect to other places.
How about using `numpy.random.RandomState(seed).permutation(len(dataset))`?